### ISSUE TITLE
safrs doesn't work in python 3.9 - TypeError

### ISSUE THREAD

**Issue by kurianbenoy-aot** [thomaxxl/safrs#83]
safrs library doesn't seems to work in Python3.9. The traceback error is shown as below:

```
Traceback (most recent call last):
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Kurian Benoy\AppData\Local\Programs\Python\Python39\Scripts\flask.exe\__main__.py", line 7, in <module>    
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\flask\cli.py", line 967, in main     
    cli.main(args=sys.argv[1:], prog_name="python -m flask" if as_module else None)
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\flask\cli.py", line 586, in main     
    return super(FlaskGroup, self).main(*args, **kwargs)
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 782, in main    
    rv = self.invoke(ctx)
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 1259, in invoke 
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 1066, in invoke 
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 610, in invoke  
    return callback(*args, **kwargs)
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\click\decorators.py", line 73, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 610, in invoke  
    return callback(*args, **kwargs)
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\flask\cli.py", line 848, in run_command
    app = DispatchingApp(info.load_app, use_eager_loading=eager_loading)
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\flask\cli.py", line 305, in __init__ 
    self._load_unlocked()
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\flask\cli.py", line 330, in _load_unlocked
    self._app = rv = self.loader()
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\flask\cli.py", line 388, in load_app 
    app = locate_app(self, import_name, name)
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\flask\cli.py", line 240, in locate_app
    __import__(module_name)
  File "D:\work\FWF292\helloanju.py", line 3, in <module>
    from safrs import SAFRSBase, SAFRSAPI
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\safrs\__init__.py", line 6, in <module>
    from .safrs_init import DB, log, SAFRS, dict_merge, test_decorator, SAFRSRequest
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\safrs\safrs_init.py", line 8, in <module>
    from .request import SAFRSRequest
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\safrs\request.py", line 18, in <module>
    from .safrs_api import HTTP_METHODS
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\safrs\safrs_api.py", line 20, in <module>
    from .json_encoder import SAFRSJSONEncoder
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\safrs\json_encoder.py", line 11, in <module>
    from .base import SAFRSBase, Included
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\safrs\base.py", line 79, in <module> 
    class SAFRSBase(Model):
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\safrs\base.py", line 329, in SAFRSBase
    def _s_relationships(cls):
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\sqlalchemy\ext\hybrid.py", line 1005, in expression
    return self._copy(expr=expr)
  File "c:\users\kurian benoy\appdata\local\programs\python\python39\lib\site-packages\sqlalchemy\ext\hybrid.py", line 918, 
in _copy
    return type(self)(**defaults)
TypeError: __init__() got an unexpected keyword argument 'cache_parameters'
```

**Comment by thomaxxl** [comment]
Hi,

I just commited a temporary workaround for this issue.
I'll update it later.

**Comment by kurianbenoy-aot** [comment]
Thanks for the quick reply @thomaxxl

**Comment by thomaxxl** [comment]
Thanks for raising this issue.
Someone brought it to my attention earlier but I didn't commit a fix yet.
I'll release an update soon.

**Comment by thomaxxl** [comment]
fixed in v2.10.6
